API-proposed.yaml

```diff
@@ -2209,6 +2209,8 @@ components:
           type: integer
           description: End time for this state
           format: int64
+      description: "If state holds a single applicable value and no style(s), that\
```

Comment:
I think this behavior is a left-over. I wasn't aware of the documentation. 

Initially it was used to way to have default behavior, if the data provider didn't provide the style. The Trace Compass server has a special code in the TimeGraphStateSerializer, the theia-trace-extension however, doesn't support that special behavior anymore (since https://github.com/eclipse-cdt-cloud/theia-trace-extension/pull/344).

We actually have a backlog task for that.

Reply:
Ack; thanks. So I'm now closing this PR alongside its abandoned companion [1] (aforementioned).